tools/onebranch/onebranch.vcxproj

```diff
@@ -102,7 +102,7 @@
       <GenerateDebugInformation>true</GenerateDebugInformation>
     </Link>
   </ItemDefinitionGroup>
-  <ItemDefinitionGroup Condition="'$(Configuration)|$(Platform)'=='Debug|x64'">
+  <ItemDefinitionGroup Condition="'$(Configuration)|$(Platform)'=='Debug|x64' And '$(BuildOneBranch)'=='True'">
```

Comment:
>And '$(BuildOneBranch)'=='True [](http://example.com/codeflow?start=78&length=30)

Why would we want the warning level and SDL checks to be gated on `BuildOneBranch`?  I think it's helpful to have them on even for local builds to not be surprised by pipeline failures when starting a PR.

Same question for the `BuildOneBranch` condition for `Release|x64` just below this

Reply:
Directory.Build.Props has SDLCheck set to true and warning level is set to 4.

Reply:
The buildOneBranch project just copies some files that is needed for a Microsoft internal build pipeline to work.

Reply:
Ah, I see.